client/dashboard/app-dotcom/style.scss

```diff
@@ -13,6 +13,7 @@
 	--wp-admin-theme-color-darker-20: #183ad6;
 	--wp-admin-theme-color-darker-20--rgb: 24, 58, 214;
 	--wp-admin-border-width-focus: 2px;
+	--wp-components-color-foreground: var(--wp-admin-theme-color);
```

Comment:
`<ProgressBar>` components use this variable, but surprisingly, we haven't set it yet.

Reply:
I know we did this for the divider component, but that one seems special core doesn't seem to have an opinion what colour that is. So we sort of have to set something.

The progress bar seems different because core _does_ set a colour for it, and so unless there's a reason to deviate, it seems like we should stick with what core does.

Of course I realise we are overriding the colour for the alert levels 😄 But I'm thinking I should contribute that upstream.